src/plus/gk/utils.ts

```diff
@@ -24,3 +25,29 @@ export async function ensurePlusFeaturesEnabled(): Promise<boolean> {
 	await configuration.updateEffective('plusFeatures.enabled', true);
 	return true;
 }
+
+export function getPreviewTrialAndDays(isDebugging: boolean) {
+	const startedOn = new Date();
+
+	let days: number;
+	let expiresOn = new Date(startedOn);
+	if (isDebugging) {
+		expiresOn = createFromDateDelta(expiresOn, { minutes: 1 });
+		days = 0;
+	} else {
```

Comment:
We can probably nuke this now and the "reset" command too?

Reply:
Good point. Going to go ahead and remove them.

Reply:
Actually, let's keep the "reset" command for subscription because it wipes the stored subscription, which is something these debug utils don't mess with. So it feels like it still has separate utility. Removing this special case date logic for preview trials seems fine though.